string state
string action
string next_state
---
float64 reward